zephyr/CMakeLists.txt

```diff
@@ -877,6 +877,17 @@ zephyr_library_sources_ifdef(CONFIG_SOF_BOOT_TEST
        boot_test.c
 )
 
+if(CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING AND NOT CONFIG_GOOGLE_RTC_AUDIO_PROCESSING_MOCK)
```

Comment:
Can this CMake addition be moved to the relevant subdirectory?
- #8630

Reply:
So, I did try.  But unfortunately no, as src/audio/google/CMakeList.txt is written to be included from src/audio/CMakeLists.txt, and that file isn't Zephyrized yet, only src/{init,ipc,module} so far.  And as much as I agree with the aim here, I just don't have the cmake patience required to make that work.

Reply:
Thanks for trying! 

> no, as src/audio/google/CMakeList.txt is written to be included from src/audio/CMakeLists.txt,

In that case can you please at least move this addition close to the existing `CONFIG_COMP_GOOGLE_RTC_AUDIO_PROCESSING / ${SOF_AUDIO_PATH}/google/google_rtc_audio_processing.c`?

It will be clearer and also  help when #8630 starts looking at SOF_AUDIO_PATH.